packages/relay/src/index.ts

```diff
@@ -60,22 +60,22 @@ export interface Eth {
 
   getBlockByHash(hash: string, showDetails: boolean): Promise<Block | null>;
 
-  getBlockByNumber(blockNum: number, showDetails: boolean): Promise<Block | null>;
+  getBlockByNumber(blockNum: string, showDetails: boolean): Promise<Block | null>;
 
   getBlockTransactionCountByHash(hash: string): Promise<number | null>;
 
-  getBlockTransactionCountByNumber(blockNum: number): Promise<number | null>
+  getBlockTransactionCountByNumber(blockNum: string): Promise<number | null>
   
   getCode(address: string, blockNumber: string | null): Promise<string>;
 
   chainId(): string;
 
-  // getLogs();
-  // getStorageAt();
+  // getLogs(fromBlock: string|null, toBlock: string|null, address: string|null, topics: any[]|null);
+  // getStorageAt(address: string, slot: string, blockNumber: string|null);
 
   getTransactionByBlockHashAndIndex(hash: string, index: number): Promise<Transaction | null>;
 
-  getTransactionByBlockNumberAndIndex(blockNum: number, index: number): Promise<Transaction | null>;
+  getTransactionByBlockNumberAndIndex(blockNum: string, index: number): Promise<Transaction | null>;
```

Comment:
now it makes sense why it was string to start

Reply:
YAGNI becomes YNI